### PR TITLE
Add 2021-01-28 meeting notes

### PR DESCRIPTION
**Pull request by matteodelabre**
Please push to this branch to add more items that should be addressed during the next meeting.

### CODE REVIEW

File: meetings/discussion-2021-01-28.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+| Name | Information |
+| ---- | ----------- |
+|      |             |
+
+# Standing Items
+
+- Determine next meeting date
+- Call for new business
+
+# Old Business
```
Comment (Eeems) [inline meetings/discussion-2021-01-28.md:40]: Should we include the format and filepath items?

Comment (matteodelabre) [reply meetings/discussion-2021-01-28.md]: Done.

File: meetings/discussion-2021-01-28.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+| [@LinusCDE](https://github.com/LinusCDE)           | [Toltec](https://github.com/toltec-dev/toltec) maintainer    |
+| [@matteodelabre](https://github.com/matteodelabre) | [Toltec](https://github.com/toltec-dev/toltec) maintainer    |
+| [@raisjn](https://github.com/raisjn)               | [remux](https://rmkit.dev/apps/remux) author and [Toltec](https://github.com/toltec-dev/toltec) maintainer |
+| [@danshick](https://github.com/danshick)           | [Toltec](https://github.com/toltec-dev/toltec) contributor   |
+
+## Regrets
+
+| Name | Information |
+| ---- | ----------- |
+| [@dixonary](https://github.com/dixonary)           | [Draft](https://github.com/dixonary/draft-reMarkable) launcher and .draft format author |
```
Comment (Eeems) [inline meetings/discussion-2021-01-28.md:33]:
```suggestion
| Name                                               | Information                                                                             |
| -------------------------------------------------- | --------------------------------------------------------------------------------------- |
| [@dixonary](https://github.com/dixonary)           | [Draft](https://github.com/dixonary/draft-reMarkable) launcher and .draft format author |
```

**Comment by LinusCDE** [comment]
LGTM. @ me when changes are done to review it again.

Sidenote, that didn't come up in the meeting: Maybe try to get [timower](https://github.com/timower/) to join the launcher specific meeting since he also has a [launcher](https://github.com/timower/rM2-stuff#launcher). He seems mentioned a few times in the [xdg-discussion](https://github.com/toltec-dev/organization/discussions/16) but not quite involved yet.

**Comment by Eeems** [comment]
> 
> 
> LGTM. @ me in Discord when changes seem done and approvals needed.
> 
> Sidenote, that didn't come up in the meeting: Maybe try to get [timower](https://github.com/timower/) to join the launcher specific meeting since he also has a [launcher](https://github.com/timower/rM2-stuff#launcher). He seems mentioned a few times in the [xdg-discussion](https://github.com/toltec-dev/organization/discussions/16) but not quite involved yet.

I can reach out and see if he wants to be part of the discussion. That said, I'm not sure he's released his launcher to a wider audience quite yet.

**Review by danshick — APPROVED** [review]
LGTM

**Review by Eeems — APPROVED** [review]

**Review by LinusCDE — APPROVED** [review]

**Review by raisjn — APPROVED** [review]

**Pull request merged**